modules/doctor/queue.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package doctor
+
+import (
+	"context"
+
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/nosql"
+	"code.gitea.io/gitea/modules/queue"
+	"code.gitea.io/gitea/modules/setting"
+	"gitea.com/lunny/levelqueue"
+)
+
+var uniqueQueueNames = []string{
```

Comment:
Now let's hope nobody forgets to update theses as we add new queues.

Reply:
I guess I'm just loathe to create tight registry of queues. Although I guess this is what this is doing.